esp/scm/ws_packageprocess.ecm

```diff
@@ -288,7 +288,7 @@ ESPresponse [exceptions_inline] GetPartFromPackageMapResponse
     string Content;
 };
 
-ESPservice [requiredaccess("NONE"), version("1.02"), exceptions_inline("./smc_xslt/exceptions.xslt")] WsPackageProcess
+ESPservice [autofeatureauth("NONE"), version("1.02"), exceptions_inline("./smc_xslt/exceptions.xslt")] WsPackageProcess
 {
```

Comment:
@afishbeck adding new features will be handled separately as sub-tasks. It should be handled by the service "owner", and you're right the new behavior needs to be coordinated with ops and perhaps other users.